app/lib/bk/compat/parsers/bitrise/github_last_commit_date.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module BK
+  module Compat
+    module BitriseSteps
+      # Handles the translation of github-last-commit-date
+      class Translator
+        def translate_github_last_commit_date(inputs)
+          raise 'Invalid github-last-commit-date step configuration!' unless inputs.key?('repository_url')
+
+          # commands = build_commands(inputs)
+          BK::Compat::CommandStep.new(commands: build_commands(inputs))
+        end
+
+        private
+
+        def build_commands(inputs)
+          repo_dir = '$(mktemp -d)'
+          commands = []
+          commands << "git clone --depth 1 #{inputs['repository_url']} #{repo_dir}"
+          commands << "cd #{repo_dir}"
```

Comment:
Don't think that a plugin should clone a repository. As with previous plugins that have this option, you should just add a comment stating that the repository is cloned by the agent